nodeup/pkg/model/kubelet.go

```diff
@@ -590,7 +592,7 @@ func (b *KubeletBuilder) buildMasterKubeletKubeconfig() (*nodetasks.File, error)
 
 	template := &x509.Certificate{
 		BasicConstraintsValid: true,
-		IsCA: false,
+		IsCA:                  false,
```

Comment:
Caused by the use of `gofmt-w`.

Reply:
Yes, this was changed in go 1.12 IIRC.  I suspect tests will fail because we're still using go 1.10 on master.  We probably need to bump the master branch to use go 1.12, although then that will cherry-picking annoying :-(

Reply:
If you just exclude this line, the rest of the PR looks good, and I bet this is what is causing the test failure...

Reply:
unexpectedly, has been updated

Reply:
I execute according to log, still be an error.
```
I0418 03:01:36.488] FAIL: 'gofmt -s -w' needs to be run on the following files:
I0418 03:01:36.488] nodeup/pkg/model/kubelet.go
I0418 03:01:36.488] FAIL: please execute make gofmt
```

Reply:
@justinsb  test passed successfully.